### DISCUSSION TITLE
Items equipped by creatures appear in 0 coordinate position

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15177] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

items equipped by monsters (for example Guardian Hook Canno and Guardian Steam Cannon) will appear in 0 coordinate position
if you go to abyss you can see a big blip on sonar

### Reproduction steps

1. make an eventset that spawns many fractal guardians
2. start a campaign and trigger monsterevents containing these creatures?
3. look at 0 position

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
![20241117234624_1](https://github.com/user-attachments/assets/874dedc7-6947-4360-aca0-8382afd3ba2d)

**Comment by ubertpendragon** [comment]
Can confirm, for a custom mission I'm spawning creatures with an item to locate them on sonar (the workarounds 😩) and the blip appears are the 0 coordinate, it eventually updates as you move but it's confusing players

Spawned at the 0 coordinate
![image](https://github.com/user-attachments/assets/f7ecd450-b126-4183-b460-8f82f7de1237)

**Comment by Regalis11** [comment]
This has been fixed in our private repo now, fix coming up in the next unstable build and the upcoming release.
